open_academy/models/open_academy_session.py

```diff
@@ -31,13 +32,20 @@ def _onchange_seats(self):
             return {
                 "warning": {
                     "title": "Incorrect values",
-                    "message": "The number of seats can not be less than 0"
+                    "message": "The number of seats can not be less than 0."
                 }
             }
-        elif len(self.attendee_ids) > self.seats:
+        if len(self.attendee_ids) > self.seats:
             return {
                 "warning": {
                     "title": "Incorrect values",
-                    "message": "The number of attendees can not be greater than the seats available"
+                    "message": "The number of attendees can not be greater than the seats available."
                 }
             }
+
+    @api.constrains("instructor_id", "attendee_ids")
+    def _check_instructor_in_attendee(self):
+        for record in self.filtered('instructor_id'):
+            __import__('pdb').set_trace()
```

Comment:
Please remove this pdb.

Reply:
hahaha sorry @edy1192 I was just playing around with that and again forgot to remove it. Thanks!